django/conf/locale/__init__.py

```diff
@@ -207,6 +207,12 @@
         "name": "French",
         "name_local": "français",
     },
+    "fr-ca": {
+        "bidi": False,
+        "code": "fr-ca",
+        "name": "Canadian French",
+        "name_local": "français canadien",
+    },
```

Comment:
```suggestion
```
This shouldn't be required for this fix, I can remove and both the automated test and manual tests work without it